src/frame.js

```diff
@@ -31,7 +31,9 @@ if (mmAppear) {
   try {
     window.ethereum = new MetaMaskProvider(new Connection())
     window.ethereum.isMetaMask = true
-    window.ethereum._metamask = true
+    window.ethereum._metamask = {
```

Comment:
did they get rid of the boolean in the interface? anything else we want to add here while we're making changes?

Reply:
I think this might've been how it was before and we just added the existence of `window.ethereum._metamask` to pass checks. 

Yeah, there might be more under this flag to mimic.